filebeat/inputsource/tcp/server.go

```diff
@@ -85,3 +86,10 @@ func (s *Server) createServer() (net.Listener, error) {
 	}
 	return l, nil
 }
+
+func (s *Server) network() string {
+	if s.config.Network != "" {
+		return s.config.Network
+	}
+	return "tcp"
```

Comment:
[Suggestion]
Use the constant you created:
```suggestion
	return networkTCP
```